ecl/data_source_ecl_network_fic_gateway_v2_test.go

```diff
@@ -0,0 +1,176 @@
+package ecl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccNetworkV2FICGatewayDataSource_basic(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip this test in short mode")
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckFICGateway(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceBasic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_1"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "description", ""),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "fic_service_id", "66a63898-32a5-4b9d-8925-f52be1d84764"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "id", "fc546cf7-1956-436b-a9b4-edc917e397cf"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "name", "F032000001492"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "qos_option_id", "d384d7f5-22aa-46e5-8cf5-759e87c7b2fd"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "status", "ACTIVE"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "tenant_id", OS_TENANT_ID),
+				),
+			},
+		},
+	})
+}
+
+func TestAccNetworkV2FICGatewayDataSource_queries(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip this test in short mode")
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckFICGateway(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
```

Comment:
こちらも型は省略できます。他のStepも同様に修正してください。
```suggestion
			{
```